SurveyApp/Modules/SurveyQuestion/Cell/ChoiceCollectionCell.swift

```diff
@@ -9,18 +9,22 @@ import UIKit
 
 class ChoiceCollectionCell: QuestionCollectionCell {
     
-    override var answerView: UIView {
+    override var answerView: AnswerView {
         get { super.answerView }
         set { return super.answerView = newValue }
     }
     
-    override func configure(with question: SurveyQuestion) {
-        if question.pickType == SurveyQuestion.PickType.any {
-            answerView = MultipleChoiceAnswerView(answers: question.sortedAnswers)
+    override func configure(with viewModel: QuestionCollectionCellViewModel) {
+        if viewModel.pickType == SurveyQuestion.PickType.any {
+            answerView = MultipleChoiceAnswerView(answers: viewModel.answers)
         } else {
-            answerView = ChoiceAnswerView(answers: question.sortedAnswers)
+            if let answerView = answerView as? ChoiceAnswerView {
+                answerView.configure(answers: viewModel.answers)
+            } else {
+                answerView = ChoiceAnswerView(answers: viewModel.answers)
+            }
         }
```

Comment:
Can you explain your intention here, please? 🙏 
Some points that I concern about:
- `MultipleChoiceAnswerView` doesn't need to call `configure`?
- Why we have 2 steps: set `answerView` in the first call and call `answerView.configure(answers: viewModel.answers)` in the second call?

Reply:
@markgravity The `UIPickerView` in `ChoiceAnswerView` will not update the data when reusing the cell, so I have to check if the `answerView` is `ChoiceAnswerView` then it needs to update the data.

Reply:
@llleyelll hmm, so create a new view doesn't work? Like this?
```swift
switch viewModel.pickType {
case .any:
  answerView = MultipleChoiceAnswerView(answers: viewModel.answers)
default:
  answerView = ChoiceAnswerView(answers: viewModel.answers)
}
```

Reply:
@markgravity it doesn't work. The next cell will still use the previous data. 😭

Reply:
@llleyelll I just try to build your app on my side, but can't reproduce your issue. Can you help me with the steps, please 🙏 

https://user-images.githubusercontent.com/17875522/125553402-5de5062a-2759-47b6-869d-d8d2bfd76a5e.mov

Reply:
@markgravity The issue occurs for some surveys. The second survey (ibis Bangkok Riverside) has two choice questions. When swipe to the second choice question it will show the first choice question's data instead of the country list in this case.
The third survey also has two choice questions but it work perfectly fine.

https://user-images.githubusercontent.com/45258998/125553971-09be3f9e-3c78-4cc7-b380-fa2e46014cb1.mov